src/components/radios/index.md.njk

```diff
@@ -84,19 +84,23 @@ If one or more of your radio options is different from the others, it can help u
 
 ### Conditionally revealing content
 
-You can conditionally reveal content when the user selects a particular radio, so they only see content when it’s relevant to them.
+You can conditionally reveal a related question when the user selects a particular radio option, so they only see content when it’s relevant to them.
 
-For example, you could reveal a phone number input when the user selects the 'Contact me by phone' option.
+This might make two related questions easier to answer by grouping them on the same page. For example, you could reveal a phone number input when the user selects the 'Contact me by phone' option.
 
 {{ example({group: "components", item: "radios", example: "conditional-reveal", html: true, nunjucks: true, open: false, size: "xl"}) }}
 
-Keep it simple. If you need to add a lot of content, consider showing it on the next page in the process instead.
+Keep it simple. If you need to ask a complicated question, consider showing it on the next page in the process instead.
 
-Do not add conditionally revealing content to inline radios.
+Do not add conditionally revealing content to inline radio options, such as ‘yes’ and ‘no’ options placed next to each other.
```

Comment:
We could make 'inline radio options' link to https://design-system.service.gov.uk/components/radios/#inline-radios. (We could also rename 'inline radio options' to 'inline radios' as we use that wording elsewhere on this page.)

Reply:
Ahh I didn't see this was being used so widely. Okay, let's say 'inline radios.'

I'd avoid the link though as it links to the same page and the style guide discourages anchor links.
```suggestion
Do not add conditionally revealing content to inline radios, such as ‘yes’ and ‘no’ options placed next to each other.
```